bigbluebutton-html5/private/config/settings.yml

```diff
@@ -1169,6 +1169,7 @@ public:
       - home.jpg
       - coffeeshop.jpg
       - board.jpg
+  # Please if editing an version here, also edit in bigbluebutton-html5/client/main.html
```

Comment:
We could consider removing these `minBrowserVersions` configs since the minimum versions are now hard-coded. It doesn’t really make sense as a configurable option if changing it requires modifying the code and rebuilding the entire client.

Also, lowering the version would cause a startup crash, so perhaps it shouldn’t be something that’s easily tweakable in the first place.

This config would only be useful if it were referenced in multiple places, which doesn’t appear to be the case.